builtin/providers/kubernetes/resource_kubernetes_config_map.go

```diff
@@ -21,6 +21,8 @@ func resourceKubernetesConfigMap() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+		Name:        "Config Map",
```

Comment:
I'm not sure we need to give resources names other than the machine-friendly names they already have. Did you have a specific use-case in mind for this? It feels to me like it would always just be a mechanical transform of the resource type name, with all the underscores replaced with spaces...